src/components/screens/secondPassphrase/confirmPassphrase.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PassphraseRenderer from '../../shared/passphraseRenderer';
+import Box from '../../toolbox/box';
+import styles from './secondPassphrase.css';
+
+const ConfirmPassphrase = ({
+  t, passphrase, nextStep, prevStep,
+}) => (
+  <Box className={styles.passphraseConfirmation}>
+    <Box.Header>
+      <h2>{t('Confirm your 2nd passphrase')}</h2>
+    </Box.Header>
+    <Box.Content>
+      <div className={styles.passphraseConfirmationContainer}>
+        <p className={styles.info}>{t('Based on your passphrase that was generated in the previous step, select the missing words below')}</p>
+        <PassphraseRenderer
+          values={passphrase.split(' ')}
```

Comment:
```suggestion
          passphrase={passphrase}
```
💅 It would be DRYer to pass `passphrase` and call `.split(' ')` inside `PassphraseRenderer` than here and  in https://github.com/LiskHQ/lisk-hub/pull/2575/files#diff-618c1dc31da4cb19645765d9e46e985eR52